xpla/chain.json

```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "xpla",
+  "status": "live",
+  "website": "https://xpla.io",
+  "network_type": "mainnet",
+  "pretty_name": "XPLA",
+  "chain_id": "dimension_37-1",
+  "bech32_prefix": "xpla",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
I register the xpla prefix on SLIP-0173